### ISSUE TITLE
Open and Browse Issues on our Discourse #2258 is really bad UX

### ISSUE THREAD

**Issue by airtonix** [Foundry376/Mailspring#2315]
discourse is an incredibly bad place to report bugs: 

- bad at linking github code
- bad at highlight code
- now i need another account just to report bugs as a developer on mailspring plugins... wut?

You should be :
- using the discussion feature of github
- using github tickets
- automating the labelling and so forth with github actions

Basically you've killed any desire to contribute to mailspring.

:clap: if that was your goal

**Comment by airtonix** [comment]
wow just wow...

looking at the discourse forum...


all you've done is close tickets here and copy them to discourse.

For someone who claims to all about inclusivity, this is a pretty isolating move.

gotta say. not impressed.

**Comment by CodeMouse92** [comment]
It's too bad you're disappointed, but we needed to make decisions that were best for the project, developers, and community as a whole, not just for a few people. None of the points you mentioned above have been a problem.

> bad at linking github code

Incorrect. We have a plugin to keep things related. We also use GitHub Issues when we start work on a task.

> bad at highlight code

Incorrect. When you highlight code and paste the link either on GitHub or Discourse, it works the same.

> now i need another account just to report bugs as a developer on mailspring plugins... wut?

Incorrect. You log in with your GitHub account.

What you described we "should be doing" was how Mailspring did it for several years. It resulted in disorganization, backlog, poor discoverability, and rendering GitHub useless to the core developer(s) who maintain this in their spare time. I cleared a 1.1K issue backlog.

> For someone who claims to all about inclusivity, this is a pretty isolating move.

I'm not sure where you get this idea. Ensuring discoverability, community feedback, and conversation can flow unimpeded, and without disrupting active development, increases engagement. We've seen that payoff already.

The full reasons for this are detailed here: https://community.getmailspring.com/t/mailspring-community-transition/38

I understand your concerns, but many weeks more thought (on top of years of experience) were used to make this decision, versus the few minutes you used to post this critique.
